priv/www/js/formatters.js

```diff
@@ -559,6 +559,14 @@ function fmt_object_state(obj) {
         colour = 'yellow';
         explanation = 'Publishing rate recently throttled by server.';
     }
+    else if (obj.state == 'terminated') {
+        colour = 'yellow';
+        var terminated_by = "";
+        if(typeof(obj.terminated_by) === "string"){
```

Comment:
Can `obj.terminated_by` be any other type?

Reply:
It may, but I prefer to be more specific with types.

Reply:
How about this then, which doesn't require a type check?

```javascript
var terminated_by = "";
if (obj.terminated_by) {
    terminated_by = " by \"" + String(obj.terminated_by) + "\""
}
explanation = 'The queue is being deleted' + terminated_by + ".";
```

Reply:
If you want to change the code - feel free.

Reply:
:+1: